source/documentation/getting_started/quick_start.md

```diff
@@ -1,10 +1,59 @@
 ## Quick start guide
 
-Registers in beta are reliable and are ready for use in products and services.
+Registers which are in beta are reliable and ready for use in products and services. Each register has an open, RESTful API which you can use to access the data in the register. 
 
-1. [Find the base URL for the register(s) you want to use](https://registers.cloudapps.digital/registers). For example, for the `local-authority-eng` register: `https://local-authority-eng.register.gov.uk/`.
-2. [Find the endpoints you want](#apireference). For example, `/record/BIR` in `https://local-authority-eng.register.gov.uk/record/BIR`.
-3. Add `.json`, `.yaml`, `.csv`, `.tsv`, or `.ttl` to the end of the request URL (before any query parameters) to choose the format of the response, or amend headers to `accept application/json` or equivalent.
-4. Parse register data and use it in your product or service.
+You can follow this guide to quickly get started with GOV.UK Registers APIs. 
+
+**1. Find the base URL for the register(s) you want to use.** 
+
+You can find a full list of register base URLs [here](https://registers.cloudapps.digital/registers).
+
+The following are some examples of base URLs:
+
+| Register name | Base URL |
```

Comment:
Technically this is the "Register ID" not the "Register name". Once the work done by @michaelabenyohai gets merged it will be more obvious.

E.g. 
Name: Local Authorities in England
ID: local-authority-eng